src/constants/depositRatio.ts

```diff
@@ -0,0 +1 @@
+export const depositRatio = 0.02;
```

Comment:
this should be set on the admin side and then passed to the frontend, not write hard code on the frontend, because it is not easy to change later, maybe you can discuss it with BA